pricing.js

```diff
@@ -7,30 +27,23 @@ module.exports.calculateProductPrice = function (product, employee, coverageLeve
       for (var i = 0; i < coverageLevels.length; i++) {
```

Comment:
By only moving the calculation piece out and not the loop, you've split the responsibility for calculating the volLife product price between two functions. A better approach is to keep all that together as a single concern in one function.